server/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type event struct {
```

Comment:
move this to a different package, also make this exportable

Reply:
Done